reana_server/config.py

```diff
@@ -264,7 +264,7 @@ def _get_rate_limit(env_variable: str, default: str) -> str:
 REANA_SCHEDULER_REQUEUE_SLEEP = float(os.getenv("REANA_SCHEDULER_REQUEUE_SLEEP", "15"))
 """How many seconds to wait between consuming workflows."""
 
-REANA_SCHEDULER_REQUEUE_COUNT = int(os.getenv("REANA_SCHEDULER_REQUEUE_COUNT", "200"))
+REANA_SCHEDULER_REQUEUE_COUNT = float(os.getenv("REANA_SCHEDULER_REQUEUE_COUNT", "200"))
```

Comment:
What about using `int(float("1e+06"))`? Just to make sure that the final result is well an integer as the main business logic expects.  

It does not make much difference here, but it could for other variables.

(We could probably use this solution for all variables where we expect an integer and where the passed value may be too huge... Unless we solve the core issue on the helm level already.)

Reply:
`int(float("1e+06"))` should work. I added `float` because you want to introduce a special value to disable REQUEUE_COUNT and float supports "infinity" as a parameter.

Reply:
If we will have some other special value like zero or -1 to disable REQUEUE, we will need to add some new logic in the code to account for that. It was just simpler with float(). 

If "infinity" is not a good name, we can have something like "disabled" or "none".

Reply:
Ah, I see; here "infinity" seems to be a handy implementation trick indeed; although "count" being "float" looks a bit weird conceptually. We can keep it here, and elsewhere for other variables (where we don't allow infinity and such) we can cast back to int.

Reply:
So to confirm, if the admin wants to disable the re-queue count check they should use "infinity" like:

```yaml
REANA_SCHEDULER_REQUEUE_COUNT: infinity
```